aspnetcore/fundamentals/minimal-apis/includes/middleware8.md

```diff
@@ -77,4 +77,55 @@ app.Run(context =>
 
 Terminal middleware is middleware that runs if no endpoint handles the request.
 
+## Antiforgery middleware
```

Comment:
I think we should put this content on [this page](https://learn.microsoft.com/en-us/aspnet/core/security/anti-request-forgery?view=aspnetcore-7.0). Replacing the "Antiforgery with Minimal APIs" heading [here](https://learn.microsoft.com/en-us/aspnet/core/security/anti-request-forgery?view=aspnetcore-7.0#antiforgery-with-minimal-apis-1).